src/scantailor/app/ui/widgets/collapsible_group_box.py

```
@@ -0,0 +1,247 @@
+
+        self._update_widgets()
+
+    def is_collapsed(self) -> bool:
+        """Return the current collapsed state.
+
+        Returns:
+            True if collapsed, False if expanded.
+        """
+        return self._collapsed
+
+    def set_collapsed(self, collapse: bool) -> None:
```

Ruff (0.15.0)

[error] Boolean-typed positional argument in function definition

(FBT001)